src/plugins/intel_npu/src/compiler/include/zero_compiler_in_driver.hpp

```diff
@@ -178,9 +178,14 @@ class LevelZeroCompilerInDriver final : public ICompiler {
 
 template <typename TableExtension>
 LevelZeroCompilerInDriver<TableExtension>::LevelZeroCompilerInDriver(const char* extName,
-                                                                     ze_driver_handle_t driverHandle)
+                                                                     ze_driver_handle_t driverHandle, ze_device_handle_t deviceHandle, ze_context_handle_t zeContext)
     : _driverHandle(driverHandle),
       _logger("LevelZeroCompilerInDriver", Logger::global().level()) {
+    
```

Comment:
Can move to init list

Reply:
yes, is it done in zero_init, so we dont need this 
i remove it in latest commit